flink-runtime/src/main/java/org/apache/flink/runtime/memory/MemoryManager.java

```diff
@@ -387,11 +387,11 @@ public void release(MemorySegment segment) {
 	 *
 	 * @param segments The segments to be released.
 	 * @throws NullPointerException Thrown, if the given collection is null.
-	 * @throws IllegalArgumentException Thrown, id the segments are of an incompatible type.
+	 * @throws IllegalArgumentException Thrown, if the segments are of an incompatible type.
 	 */
 	public void release(Collection<MemorySegment> segments) {
 		if (segments == null) {
-			return;
+			throw new NullPointerException();
 		}
 
```

Comment:
Why should it throw NullPointerException? Besides, I think a test case is needed.

Reply:
> Why should it throw NullPointerException? Besides, I think a test case is needed if it should throw NullPointerException.

Hi chunweilei, thanks a lot for your comments. The Javadoc for this method says "@throws NullPointerException Thrown, if the given collection is null."
So an NPE should be thrown, or this comment should be removed.
According to the style of the other part of the code, I think throwing an NPE is preferred.

Reply:
Sounds reasonable. Thanks for your kindly reply.